net-misc/moonlight/moonlight-9999.ebuild

```diff
@@ -51,15 +42,8 @@ RDEPEND="
 	wayland? ( dev-libs/wayland )
 	X? ( x11-libs/libX11 )
 "
-
-DEPEND="
-	${RDEPEND}
-"
-
-BDEPEND="
-	dev-qt/qtcore
-	virtual/pkgconfig
-"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
```

Comment:
I guess this was wrong before, but you probably need `dev-qt/qtbase:6` in `BDEPEND` for moc.